### ISSUE TITLE
Loss collapse during training 

### ISSUE THREAD

**Issue by aisagarw** [leftthomas/SimSiam#1]
I am trying to pretrain the SimSiam model on mscoco dataset.. but the loss collapses to -1 very quickly.. What are the possible reasons behind and some suggestions to solve the same?

**Comment by leftthomas** [comment]
@aisagarw try to use larger batch size and smaller learning rate.
